setup.py

```diff
@@ -8,7 +8,7 @@
     'moto==1.3.16',
     'mock==2.0.0',
     'pytest==3.10.1',
-    'flake8==3.8.4'
+    'flake8==4.0.0'
```

Comment:
We should add a comment here as to why this is pinned at this version so future us doesn't try to update it.